docs/03-customization.md

```diff
@@ -0,0 +1,48 @@
+# Customizations
+
+You can configure some features of balenaSound by using environment variables. This can be set in the balena dashboard: 
+```
+navigate to dashboard -> your app -> Environment variables. 
+```
+
+![Setting the device name](https://raw.githubusercontent.com/balenalabs/balena-sound/master/images/device-name-config.png)
+
+You can read more about environment variables [here](https://www.balena.io/docs/learn/manage/serv-vars/#fleet-environment-and-service-variables).
+
+## Change device name
+
+By default, your device will be named `balenaSound xxxx`. This name will show within Airplay device lists, for Spotify Connect and when searching for devices using Bluetooth.
```

Comment:
"This name will show within Airplay device lists, for Spotify Connect, and when searching for devices using Bluetooth."